pyfakefs/fake_io.py

```diff
@@ -102,7 +102,7 @@ def open(
                 for sn in self.skip_names
             ]
         ):
-            return io.open(
+            return io.open(  # pytype: disable=wrong-arg-count
```

Comment:
I have no idea where this comes from from - for some reason, it doesn't expect the last argument (`opener`).